src/Documentation/grains/actor_identity.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: page
+title: Actor Identity
```

Comment:
I think we should change the name to Grain identity and generally replace 'actor' with 'grain' in most/all docs (except in places where we're specifically talking about the Actor Model, if there are any)

Reply:
Maybe @sergeybykov has an opinion on this.

"virtual actor" would become "grain"

Reply:
I have renamed the file in one PR and this change will be made throughout this document. 
I will seek and change this in other files, too. 

Thanks, @ReubenBond !